app/index.html

```diff
@@ -39,6 +40,10 @@
     <script src="bower_components/ion-datetime-picker/release/ion-datetime-picker.min.js"></script>
     <script src="bower_components/moment/moment.js"></script>
     <script src="bower_components/angular-messages/angular-messages.js"></script>
+    <script src="bower_components/sifter/sifter.js"></script>
+    <script src="bower_components/microplugin/src/microplugin.js"></script>
+    <script src="bower_components/selectize/dist/js/selectize.js"></script>
+    <script src="bower_components/angular-selectize2/dist/angular-selectize.js"></script>
```

Comment:
@lglopezf sírvase

https://github.com/selectize/selectize.js#installation-and-files

<img width="797" alt="screen shot 2016-12-02 at 11 47 48 pm" src="https://cloud.githubusercontent.com/assets/1407000/20856878/d24b881a-b8e9-11e6-99b5-1ffdf6968ca5.png">